docs/source/release/v6.8.0/Muon/FDA/Bugfixes/35599.rst

```diff
@@ -0,0 +1,2 @@
+- Fix typo of assert in DualPlotMaxentPanePresenterTest.test_clear()
```

Comment:
I don't think we need a release note because it will be invisible to the end user.